### DISCUSSION TITLE
Repositories do not have onboarding PR

### DISCUSSION THREAD

**Discussion by kevin-matthews-vc** [renovatebot/renovate#27582] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

I enabled interactive mode on some repos within mend, this worked fine. 

I have now enabled interactive mode across the entire org, some repos have the onboarding PR, some don't. How can I force all of them to create?

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
It's possible to individually go to repos and click this:

<img width="307" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/75e667f0-78dc-4969-b7e8-55f59191a2f8">

But I'd like if you can tell us the org name so we can debug on our backend why it didn't trigger jobs automatically. I'm also curious if it might have eventually enqueued jobs too (either ever 4 hours or 24 hours are when scheduled jobs are done)

**Reply by kevin-matthews-vc** [reply]
organisation name is '.. snip ..'

**Reply by rarkins** [reply]
Thanks, I'll ask the backend team to take a look. It's possible that nobody wrote a "trigger" so that when you change to org to interactive all the non-onboarded repos inside it get a job enqueued.

**Reply by kevin-matthews-vc** [reply]
thanks, it seems some did get it - but most didn't, so it also might be rate limit based

**Reply by kevin-matthews-vc** [reply]
Hi, is there any more on this? we still have many repos without any onboarding PRs.
Thanks

**Reply by rarkins** [reply, marked as answer]
"Automatically running on every repo if the org default changes to Interactive" is not yet supported in the backend. As a workaround you can trigger runs per-repo from within the UI. Once a repo is run once, it should then be run automatically thereafter

**Reply by kevin-matthews-vc** [reply]
ah ok.. there are about 95 repos :( thanks for looking in to it.

**Reply by kevin-matthews-vc** [reply]
I have manually been through all now to enable them, and altho the logs say there is no onboarding PR... there are no errors shown in the logs but the onboarding PR is still not available. 

I am happy to go through and manually add the renovate.json files with the correct contents, but is this the right behaviour?

**Reply by rarkins** [reply]
I don't understand what you've written, please elaborate on what you mean

**Reply by kevin-matthews-vc** [reply]
It's ok I expected an onboarding pr to be in all repos but that wasn't the case I've since found some repos don't have dependencies so I assume renovate recognises that and doesn't create an onboarding pr

**Reply by rarkins** [reply]
Yes, Renovate skips onboarding repos where it can't find dependencies
